<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClassOverEvaluate</class>
 <widget class="C8CommonWindow" name="ClassOverEvaluate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ClassOverEvaluate</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="widget_classoverEvaluate_bk" native="true">
     <property name="minimumSize">
      <size>
       <width>403</width>
       <height>322</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>14</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>24</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>16</number>
        </property>
        <item>
         <widget class="QWidget" name="widget_titleBar" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="rightMargin">
              <number>5</number>
             </property>
             <item>
              <widget class="QLabel" name="label_classEvaluation">
               <property name="text">
                <string>classEvaluation</string>
               </property>
               <property name="indent">
                <number>16</number>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_close">
               <property name="minimumSize">
                <size>
                 <width>27</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>27</width>
                 <height>30</height>
                </size>
               </property>
               <property name="toolTip">
                <string>close</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
          <zorder></zorder>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_timesUp_classOver">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string>timesUp_classOver</string>
            </property>
            <property name="indent">
             <number>18</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>12</number>
          </property>
          <property name="leftMargin">
           <number>18</number>
          </property>
          <item>
           <widget class="QLabel" name="label_Composite">
            <property name="text">
             <string>Composite</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="EvaluateButton" name="pushButton_evaluateButton">
            <property name="minimumSize">
             <size>
              <width>95</width>
              <height>15</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>95</width>
              <height>15</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_evaluate">
            <property name="text">
             <string>star_5</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>12</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="leftMargin">
           <number>14</number>
          </property>
          <property name="rightMargin">
           <number>14</number>
          </property>
          <item>
           <widget class="EvaluationEdit" name="textEdit_evaluate">
            <property name="minimumSize">
             <size>
              <width>364</width>
              <height>112</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>364</width>
              <height>112</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_TextCount">
            <property name="text">
             <string>TextCount</string>
            </property>
            <property name="indent">
             <number>24</number>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>12</number>
            </property>
            <property name="rightMargin">
             <number>14</number>
            </property>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_Cancel">
              <property name="text">
               <string>close</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_OK">
              <property name="minimumSize">
               <size>
                <width>72</width>
                <height>26</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>72</width>
                <height>26</height>
               </size>
              </property>
              <property name="text">
               <string>send</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>C8CommonWindow</class>
   <extends>QWidget</extends>
   <header>c8commonwindow.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>EvaluateButton</class>
   <extends>QPushButton</extends>
   <header location="global">evaluatebutton.h</header>
  </customwidget>
  <customwidget>
   <class>EvaluationEdit</class>
   <extends>QTextEdit</extends>
   <header location="global">evaluationedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_OK</sender>
   <signal>clicked()</signal>
   <receiver>ClassOverEvaluate</receiver>
   <slot>submitAndSignoutBtnClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>386</x>
     <y>320</y>
    </hint>
    <hint type="destinationlabel">
     <x>420</x>
     <y>332</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_close</sender>
   <signal>clicked()</signal>
   <receiver>ClassOverEvaluate</receiver>
   <slot>closeBtnClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>400</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>420</x>
     <y>4</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>submitAndSignoutBtnClicked()</slot>
  <slot>closeBtnClicked()</slot>
 </slots>
</ui>
